index.html

```diff
@@ -41,442 +19,942 @@
+          "
+        ></div>
+        <div
+          class="item"
+          style="
+            background-image: url('assets/images/intro-carousel/pattern1.jpg');
+          "
+        ></div>
+        <div
+          class="item"
+          style="
+            background-image: url('assets/images/intro-carousel/pattern2.jpg');
+          "
+        ></div>
+        <div
+          class="item"
+          style="
+            background-image: url('assets/images/intro-carousel/pattern3.jpg');
+          "
+        ></div>
+      </div>
+    </section>
+    <!-- #intro -->
+
+    <main id="main">
+      <!--==========================
+      About Section
+    ============================-->
+      <section class="about wow fadeInUp bg-black">
+        <div class="container">
+          <div class="row">
+            <div class="col-lg-6 about-img">
+              <img src="assets//images/turung-logo.png" alt="" />
             </div>
-            <p>
-              * Fast and reliable multiplatform cash-out option e.g., piggy
-              vest, Visa, MasterCard,
-            </p>
-          </div>
 
-          <div class="benefit benefit-two">
-            <div class="benefit-img">
-              <img src="./images/persons.jpg" alt="persons" />
+            <div class="col-lg-6 content">
+              <div class="owl-carousel portfolio-carousel">
+                <div class="portfolio-div">
+                  <h2 class="text-center fnt-darker-grey">
+                    The Future of commerce is NOW!
+                  </h2>
+                  <h3 class="fnt-light-grey">Imagine the impossible</h3>
+
+                  <ul class="fnt-dark-grey">
+                    <li>
+                      <i class="ion-android-checkmark-circle"></i>Online is
+                      already becoming the most preferred place to feature goods
+                      and services and experts had estimated that by the year
+                      2040, 95% of purchases will be through e-commerce
+                      platforms. Currently, traditional brick and mortar stores
+                      have more off-line presence in rural and urban areas
+                      compared with their online presence. Turung is designed to
+                      provide free online stores to enable off-line stores to
+                      sell their products or services beyond their physical
+                      location and local currency
+                    </li>
+                  </ul>
+                </div>
+                <div class="portfolio-div">
+                  <h2 class="text-center fnt-darker-grey">Turung!</h2>
+                  <h3 class="fnt-light-grey">
+                    With Turung Smartstore your store doesn't just sell, it
+                    becomes a mini-bank!
+                  </h3>
+
+                  <ul class="fnt-dark-grey">
+                    <li>
```

Comment:
![image](https://user-images.githubusercontent.com/60205816/179095852-f881bf22-058e-4d94-8636-da6b8cae3022.png)

I think you omitted the read-more button that was added to long paragraphs on mobile screens. Kindly add them back if possible.

Reply:
Read more couldnt be applied completely on carousel.

Reply:
added read more button on other long text

Reply:
Okay, that's fine. I saw the paragraphs where the read-more button was added. 
I think we can work with that :smiley: